packages/@n8n/nodes-langchain/nodes/vendors/OpenAi/helpers/error-handling.ts

```diff
@@ -23,7 +24,10 @@ export const openAiFailedAttemptHandler = (error: any) => {
 			const customErrorMessage = getCustomErrorMessage(errorCode);
 
 			if (customErrorMessage) {
-				error.message = customErrorMessage;
+				if (error.error) {
```

Comment:
What is this `error.error`? Is it used for something?

Reply:
It is wrapped original error, which we recursivly traverse for error.message too